test/mc_forecast/events_test.rb

```diff
@@ -63,4 +63,53 @@ def test_sums_rand
     assert_in_delta(4, e[:rand][:sum][:quantiles][0.025], 0.3)
     assert_in_delta(8, e[:rand][:sum][:quantiles][0.975], 0.3)
   end
+
+  def test_ranges_zero
+    n_steps = 22
+    ranges = [0..9, 10..21]
+    e = McForecast::Simulation.new.run(steps: n_steps, ranges: ranges) do |_state, _step, _trial|
+      events = {}
+      events[:zero] = 0
+
+      [nil, events]
+    end
+    assert_in_delta(0, e[:zero][:ranges][ranges[0]][:mean], 0.03)
```

Comment:
if the ranges[] are indexed with the number like this then we should update the comment in simulation.rb:28